### PR TITLE
Added JS for "Book this room buttons"

### PR DESCRIPTION
**Pull request by MickeMarten**
Added JS-code for "Book this room buttons".

Problems: Cant use css, Nelson the brave has to explain how he works with Sass, therefor no css.
                 When classs called to JavasScript only one button on the challangeCard works.

No problems: It just works. Elements are created inside Dom.

### CODE REVIEW

**Comment by FelixHultman** [comment]
looks good.

**Comment by MickeMarten** [comment]
> looks good.

Thanks

File: .vscode/settings.json
Comment (FelixHultman) [inline .vscode/settings.json:1]: is this needed?

**Review by FelixHultman — COMMENTED** [review]
Damn bro, working really hard on a friday night! i love it!

**Pull request merged**